MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2452,10 +2462,8 @@ protected List<FileSystemMetadata> GetLocalMetadataFilesToDelete()
             }
 
             var filename = System.IO.Path.GetFileNameWithoutExtension(Path);
-            var extensions = new List<string> { ".nfo", ".xml", ".srt", ".vtt", ".sub", ".idx", ".txt", ".edl", ".bif", ".smi", ".ttml" };
-            extensions.AddRange(SupportedImageExtensions);
```

Comment:
why this is no longer added to `extensions`?

Reply:
It is